gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/model/GbGradeInfo.java

```diff
@@ -40,6 +40,9 @@
 	@Getter
 	private final String gradeComment;
 
+	@Getter
+	private final boolean excuse;
```

Comment:
rename to "excused" for consistency in method names? ie. GbGradeInfo.isExcused(), GradeDefinition.isExcused()